storage/excel/datasource.go

```diff
@@ -110,13 +110,13 @@ func (ds *DataSource) Get(ctx context.Context, variables octosql.Variables, stre
 		batchSize:        ds.batchSize,
 	}
 
-	go func() {
-		log.Println("worker start")
-		rs.RunWorker(ctx)
-		log.Println("worker done")
-	}()
-
-	return rs, execution.NewExecutionOutput(execution.NewZeroWatermarkGenerator(), map[string]execution.ShuffleData{}), nil
+	return rs,
+		execution.NewExecutionOutput(
+			execution.NewZeroWatermarkGenerator(),
+			map[string]execution.ShuffleData{},
```

Comment:
I don't know which one is more elegant, but this `map[string]execution.ShuffleData{}` appears in multiple places - maybe just define `EmptyShuffleData` and use it